packages/wp-now/README.md

```diff
@@ -208,92 +274,51 @@ For example, if you run `wp-now start --blueprint=path/to/blueprint-example.json
--   `wp-now` works across all platforms (Mac, Linux, Windows);
--   `wp-now` does not support custom domains or SSL (yet!);
--   `wp-now` works with WordPress themes and plugins even if you don't have WordPress installed;
--   `wp-now` allows to easily switch the WordPress version with `wp-now start --wp=version.number`(make sure to replace the `version.number` with the actual WordPress version);
--   `wp-now` does not support Xdebug PHP extension (yet!)
-
-Some similarities between Laravel Valet and `wp-now` to be aware of:
+Here's what you need to know if you're migrating from Laravel Valet:
 
--   could be used for non-WordPress projects;
--   deployments are not possible with neither Laravel Valet, nor `wp-now`;
--   possible to switch easily the PHP version;
--   possibility to work on multiple WordPress sites simultaneously
+-   `wp-now` handles the entire WordPress installation for you. Just run the `wp-now start` command.
+-   `wp-now` works across all desktop platforms (Mac, Linux, Windows).
+-   `wp-now` does not set up custom domains for you.
+-   `wp-now` works with WordPress themes and plugins even if you don't have WordPress installed (see item #1 above).
+-   `wp-now` allows you to switch the WordPress version with `wp-now start --wp=VERSION_NUMBER`(replace `VERSION_NUMBER` with the actual WordPress version).
 
 ### wp-env
 
-If you are migrating from `wp-env`, you should be aware of the differences it has with `wp-now`:
+Here's what you need to know if you're migrating from `wp-env`:
 
--   `wp-now` supports non-WordPress projects;
--   `wp-now` does not need Docker;
--   `wp-now` does not support Xdebug PHP extension (yet!);
--   `wp-now` does not include Jest for automatic browser testing
-
-Some similarities between `wp-env` and `wp-now` to be aware of:
-
--   no support for custom domains or SSL;
--   `plugin`, `themes` and index modes are available on `wp-env` and `wp-now`;
--   deployments are not possible with neither `wp-env`, nor `wp-now`;
--   possible to switch easily the PHP version
+-   `wp-now` supports non-WordPress projects.
+-   `wp-now` does not require Docker.
+-   `wp-now` does not include Jest for automatic browser testing.
 
 ## Contributing
 
 We welcome contributions from the community!
 
-In order to contribute to `wp-now`, you'll need to first install a few global dependencies:
+### Code contribution
 
--   Make sure you have `nvm` installed. If you need to install it first,
-    [follow these installation instructions](https://github.com/nvm-sh/nvm#installation).
--   Install `nx` by running `npm install -g nx`.
-
-Once the global dependencies are installed, you can start using the repo:
+Clone the repository, install the dependencies, build the project, and start `wp-now` in `preview` mode:
 
 ```bash
 git clone git@github.com:WordPress/playground-tools.git
 cd playground-tools
-nvm use
 npm install
 npm run build
-nx preview wp-now start --path=/path/to/wordpress-plugin-or-theme
```

Comment:
If we remove the `--path` , let's at least mention it below. It's super handy for testing wp-now in different modes.

Reply:
@sejas, Added it below (see https://github.com/WordPress/playground-tools/pull/251/commits/b882d21dfe639280fa297d898960466c6adaa022)